bindings/bind_py_module.cc

```diff
@@ -38,21 +38,27 @@ PYBIND11_MODULE(_rascal, mod) {
 
   py::module m_rpr_mng = mod.def_submodule("RepresentationManager");
   m_rpr_mng.doc() = "Representation Manager Classes";
-  py::module m_feat_mng = mod.def_submodule("FeatureManager");
-  m_feat_mng.doc() = "Feature Manager Classes";
+  py::module m_models = mod.def_submodule("Models");
+  m_models.doc() = "Collection of models";
+
+  py::module m_kernels = m_models.def_submodule("Kernels");
+  m_kernels.doc() = "Collection of Kernels";
 
   py::module m_utl = mod.def_submodule("utils");
   py::module m_math = mod.def_submodule("math");
   m_math.doc() = "Collection of math functions";
   py::module m_throwaway = mod.def_submodule("rubbish");
```

Comment:
```suggestion
   py::module m_throwaway = mod.def_submodule("_utils");
```
and I would suggest replacing the name `m_throwaway` with `m_internal` throughout.